pkg/pvcresizing/pvc.go

```diff
@@ -1,9 +1,14 @@
 package pvcresizing
 
+import "strconv"
+
 const (
 	PrometheusStorageSizeSmall  PrometheusStorageSizeType = "small"
 	PrometheusStorageSizeMedium PrometheusStorageSizeType = "medium"
 	PrometheusStorageSizeLarge  PrometheusStorageSizeType = "large"
+
+	// We apply a ratio to the volume storage size to compute the RetentionSize property (RetentionSize = 90% volume storage size)
+	VOLUME_STORAGE_LIMIT = 0.9
```

Comment:
Would be nice to have this in keys, like these: https://github.com/giantswarm/prometheus-meta-operator/blob/master/service/key/key.go#L33